build-tools/packages/version-tools/src/schemes.ts

```diff
@@ -20,15 +24,22 @@ import { isInternalVersionRange, isInternalVersionScheme } from "./internalVersi
  *
  * - "internal" is the 2.0.0-internal.1.0.0 scheme.
  *
+ * - "internalPrerelease" is the 2.0.0-internal.1.0.0.[CI build #] scheme.
+ *
  * - "virtualPatch" is the 0.36.1002 scheme.
  */
-export type VersionScheme = "semver" | "internal" | "virtualPatch";
+export type VersionScheme = "semver" | "internal" | "internalPrerelease" | "virtualPatch";
```

Comment:
We do not have public / non-internal prereleases ?

Reply:
This type represents the "scheme" used. This distinction is needed because versions using the internal scheme are always pre-release according to semver. Partners need to be able to distinguish between "releases" using the new version scheme, and "prereleases" using the internal scheme. We do publish pre-releases to internal feeds, and partners get their list of packages from those feeds.